bin/README.md

```diff
@@ -0,0 +1,94 @@
+Customized interpreter for Alk
+==============================
+
+Requirements:
+-------------
+
+* The K framework - use the version of K indicated [here](https://github.com/alk-language/k-semantics).
+  It is highly recommended to put the paths to K's binaries in your system's PATH.
+
+How to use?
+-----------
+
+1. Compile the definition Alk that resides in the `alk` directory of this repo:
+
+```> kompile alk.k```
+
+or 
+
+```> pathToK/bin/kompile alk.k```
+
+This will generate a file called `alk-kompiled` in the same directory.
+
+2. Add `bin` (the folder where this README file resides) into your system's PATH.
+
+3. Run `alk` (or `alk.exe` on Windows) for the first time:
+
+```> alk```
+
+and you'll get:
+```
+Please provide an Alk file.
+Usage:
+alk(.exe) [-dhiks] [long options...] <file.alk>
+        -i STR --init STR         the initial Alk state
```

Comment:
Someone non-familiar with the option system cannot understand how to use it. We have to explicitly write that she has to choose between --i STR and --init STR.